rawJob.go

```diff
@@ -1,21 +1,21 @@
 package beanstalkworker
 
-import "strconv"
 import "time"
 import "github.com/kr/beanstalk"
 import "fmt"
 import "log"
 
-// PrioLow is the priority that jobs are released/buried with.
-const PrioLow = 1025
```

Comment:
@tomponline Does it mean that we have to set up the priority in all calling code before calling Bury and Release method? If yes, good to keep that in mine as we are updating codes to use your beanstalworker. I think that the current behavior was using the constant PrioLow.